tutorials/flaml-tutorial-pydata-23.md

```diff
@@ -0,0 +1,64 @@
+
+## Lab Forum Slides (To Be Added)
+
+## What Will You Learn?
+
+
+In this session, we will provide an in-depth and hands-on tutorial on Automated Machine Learning & Tuning with a fast python library named FLAML. We will start with an overview of the AutoML problem and the FLAML library. We will then introduce the hyperparameter optimization methods empowering the strong performance of FLAML. We will also demonstrate how to make the best use of FLAML to perform automated machine learning and hyperparameter tuning in various applications with the help of rich customization choices and advanced functionalities provided by FLAML. At last, we will share several new features of the library based on our latest research and development work around FLAML and close the tutorial with open problems and challenges learned from AutoML practice.
+
+In this session, we will provide an in-depth and hands-on tutorial on Automated Machine Learning & Tuning with a fast python library FLAML. AutoML is playing an increasingly impactful role in modern machine learning and artificial intelligence lifecycles. According to a recent report from ReportLinker, the AutoML market is predicted to increase from $346.2 million in 2020 to $14,830.8 million in 2030, demonstrating a compound annual growth rate of 45.6%[1].
+
+FLAML[2] started as a research project in Microsoft Research, and has grown to a popular open-source library. It has accumulated over 2k stars and about 800k downloads since its first release in December 2020. Compared to other AutoML tools, FLAML is notable for being fast, economical, and easy to customize. FLAML can increase the productivity of ML and data science practitioners while getting superior model prediction performance. The flexibility and customizability make it a powerful tool for R&D.
+
+In this session, we will give a hands-on tutorial on (1) how to use FLAML to automate typical machine learning tasks and generic tuning on user-defined functions; (2) how to make the best use of FLAML to perform AutoML and tuning in various applications with the help of rich customization choices and advanced functionalities provided by FLAML; and (3) several new features of FLAML based on our latest research and development work around FLAML.
+
+Target audience and prerequisites. The primary target audience of this tutorial is machine learning and data science practitioners, including data scientists, machine learning engineers, domain experts, and researchers, especially researchers who are interested in the automation of machine learning or artificial intelligence methods. The audiences are assumed to have the basic knowledge of machine learning, data science, and Python programming language.
+
+Strategies and needs of the hands-on tutorial. This tutorial is intended to be hands-on and interactive. Most of the topics covered in the tutorial will be accompanied by demonstrations implemented in Jupyter notebooks.
+
+### Tutorial Outline
```

Comment:
The tutorial outline is updated, what about we wait two more days and update this part according to the final presentation?